package.json

```diff
@@ -155,7 +155,7 @@
     "iframe-stream": "^3.0.0",
     "inject-css": "^0.1.1",
     "jazzicon": "^1.2.0",
-    "json-rpc-engine": "^3.7.3",
+    "json-rpc-engine": "github:metamask/json-rpc-engine",
```

Comment:
Alternatively could we publish to `@MetaMask/json-rpc-engine`? If this is the easiest option could we at least use a commit or tag?

Reply:
If this is temporary I can understand not wanting to update usages

Reply:
Yeah, this is totally stopgap since we don't have permissions to land a PR against or release kumavis/json-rpc-engine. As soon as we do, we'll swap this out with a proper upstream release. This MetaMask-owned fork was created solely for this PR so its `master` should never be pushed to after this bugfix. We should really delete this fork altogether to avoid confusion once we're back on the canonical package.